cluster-autoscaler/cloudprovider/cloud_provider.go

```diff
@@ -86,6 +86,13 @@ const (
 	RancherProviderName = "rancher"
 )
 
+// GpuConfig contains the label, type and the resource name for an accelerator (e.g. gpu)
```

Comment:
nit: I think there's one "accelerator" leftover in the comment here

Reply:
Done :) Removed accelerator from the comment and replaced it with GPU